core/runtime/src/test/java/io/quarkus/runtime/configuration/ConfigProfileTestCase.java

```diff
@@ -25,21 +25,17 @@
 public class ConfigProfileTestCase {
 
     static ClassLoader classLoader;
-    static ConfigProviderResolver cpr;
+    static ConfigProviderResolver configProviderResolver;
 
     @BeforeAll
     public static void initConfig() {
         classLoader = Thread.currentThread().getContextClassLoader();
-        cpr = ConfigProviderResolver.instance();
+        configProviderResolver = ConfigProviderResolver.instance();
```

Comment:
I think we can drop all of the Config registration and just use SmallRyeConfig locally at each test method.